### DISCUSSION TITLE
Can you add posibility to serve banner to specific domain?

### DISCUSSION THREAD

**Discussion by tavoweb** [giuliopons/adadmin#110] [Q&A]
Are its possible that all spaces used on all domains and in banner section then chusing advertismen select possible place and filter on wich domain the advertisment be showed ??

**Comment by giuliopons** [comment]
Can you try to explain your request better? At this momenti isn't possible to limit the views on a domain if you put the Position script in different domains.

**Comment by tavoweb** [comment]
The explanation would be as follows:
The ability to insert advertising positions into multiple website advertising spaces, for example at the top of the page 728x60, and one and the same code is inserted, for example, into domain.com and domain2.com and domain3.com. The client creates a campaign and during the campaign selects domain.com, creates a banner for his advertisement and the code automatically advertises the banner on the selected domain.com but does not advertise domain2.com and domain3.com

**Comment by giuliopons** [comment]
I think I've already added this feature in one of the past updateds:

<img width="1613" height="903" alt="image" src="https://github.com/user-attachments/assets/9e6a4ff3-18cf-4041-9f50-d02a04e9389a" />

This is a filter on the referrer which is the site that host the position script, in the above example it shows the banner only if it is on domain2.com.

**Comment by tavoweb** [comment]
perfect.
